android/src/main/java/com/tailscale/ipn/ui/model/Ipn.kt

```diff
@@ -16,20 +16,14 @@ class Ipn {
                 NeedsMachineAuth(3),
                 Stopped(4),
                 Starting(5),
-                Running(6),
-        }
+                Running(6);
 
-        // NotifyWatchOpt is a bitmask of options supplied to the notifier to specify which
-        // what we want to see on the Noitfy bus
-        enum class NotifyWatchOpt(val value: Int) {
-                engineUpdates(0),
-                initialState(1 shl 1),
-                prefs(1 shl 2),
-                netmap(1 shl 3),
-                noPrivateKeys(1 shl 4),
-                initialTailFSShares(1 shl 5)
+                companion object {
+                        fun fromInt(value: Int): State? {
+                                return State.values().first { s -> s.value == value }
```

Comment:
State.values().firstOrNull { it.value == value } instead? otherwise it'll throw an exception if no match is found